05 - Concepts/Dont ask to ask.md

```diff
@@ -0,0 +1,20 @@
+---
+aliases: 
+- 
+tags:
+- seedling
+publish: true
+---
+
+# Don't ask to ask, just ask
+
```

Comment:
Am trying to read this as someone who has not used this same phrase myself online...

I think people might reasonably ask why this page exists, and who it is aimed at.

The intro on [https://dontasktoask.com](https://dontasktoask.com) says:

> Every now and then, in online chat rooms I hang around in, someone pops in and says something in the lines of,

So I wonder about something like:

> Every now and then, in the Obsidian Members Group Discord server, someone pops in and says something in the lines of:

Reply:
well, the dont-ask-to-ask principle does work also holds outside of the Obsidian Discord server, so I phrased it mroe generally.